packages/components/table/src/store/helper.ts

```diff
@@ -12,7 +12,6 @@ const InitialStateMap = {
   selectOnIndeterminate: 'selectOnIndeterminate',
   indent: 'indent',
   lazy: 'lazy',
-  data: 'data',
```

Comment:
I noticed that some places are using the `store.states.data`. won't this modification cause any problems?

[table.vue#L16-L17](https://github.com/element-plus/element-plus/blob/dev/packages/components/table/src/table.vue#L16-L17)